AssetManagementTestProject/DAO/UserDAO.cs

```diff
@@ -81,10 +80,16 @@ public class UserDAO
     /// </summary>
     /// <param name="userName"></param>
     /// <param name="password"></param>
-    public UserDAO(string userName, string password)
+    public UserDAO(string userName, string password, string token, string id)
     {
-        UserName = userName;
-        Password = password;
+        this.UserName = userName;
+        this.Password = password;
+    }
+    public string username { get; set; }
+    public string password { get; set; }
+    public UserDAO(string username, string password)
+    {
+        this.username = username;
+        this.password = password;
     }
```

Comment:
This is confusing: For the above code, you created a UserDAO object with params token + string but never initialize them? And here you created another UserDAO object to do what?